File: translations/codenet/deepseek_commented/C2Go/starcoder/Code/s401511324.go
<<<<<<< HEAD
=======
package main
>>>>>>> 98c87cb78a (data updated)

import (
	"fmt"
	"os"
	"sort"
	"strconv"
	"strings"
)

func main() {
	var n int
	var l []int
	var i, j, k, cnt, tmp int
	// Read the number of elements in the array
	fmt.Scanf("%d", &n)
	// Read the elements of the array
	for i = 0; i < n; i++ {
		fmt.Scanf("%d", &l[i])
	}
	// Sort the array in ascending order using bubble sort
	for i = 0; i < n-1; i++ {
		for j = i + 1; j < n; j++ {
			if l[j] < l[i] {
				tmp = l[i]
				l[i] = l[j]
				l[j] = tmp
			}
		}
	}
	// Count the number of valid triangles that can be formed
	for i = 0; i < n-2; i++ {
		for j = i + 1; j < n-1; j++ {
			for k = j + 1; k < n; k++ {
				if l[i]+l[j] > l[k] {
					cnt++
				} else {
					break
				}
			}
		}
	}
	// Print the count of valid triangles
	fmt.Printf("%d\n", cnt)
}

File: translations/codenet/uncommented/C2P/codellama/Code/s401511324.py
#include<stdio.h>
int main():
  int n,l[2001],i,j,k,cnt=0,tmp=0;
  scanf("%d",&n);
  for i in range(0,n):
    scanf("%d",&l[i]);
  for i in range(0,n-1):
    for j in range(i+1,n):
      if l[j]<l[i]:
        tmp=l[i];
        l[i]=l[j];
        l[j]=tmp;
  for i in range(0,n-2):
    for j in range(i+1,n-1):
      for k in range(j+1,n):
        if l[i]+l[j]>l[k]:
          cnt++;
        else:
          break;
  printf("%d\n",cnt);
  return 0;
